.changeset/cuddly-avocados-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"changelog": minor
+---
+
+Add a Rust Cursor API and refactor the Rust Visitor API to run on top of it. Add first pass of Typescript binding to the Cursor API, but no TS Visitor yet. Add TokenNode.text to the TS API.
```

Comment:
I wonder if this should split into three entires (3 changesets)?

- Add a Rust Cursor API and refactor the Rust Visitor API to run on top of it.
- Add first pass of Typescript binding to the Cursor API, but no TS Visitor yet.
- Add `TokenNode.text` to the TS API.

Reply:
done